src/sqlfluff/dialects/tsql_keywords.py

```diff
@@ -189,4 +189,44 @@
     "WRITETEXT",
     "EXIT",
     "PROC",
+    "DATEFIRST",
+    "DATEFORMAT",
+    "DEADLOCK_PRIORITY",
+    "LOCK_TIMEOUT",
+    "CONCAT_NULL_YIELDS_NULL",
+    "CURSOR_CLOSE_ON_COMMIT",
+    "FIPS_FLAGGER",
+    "IDENTITY_INSERT",
+    "LANGUAGE",
+    "OFFSETS",
+    "QUOTED_IDENTIFIER",
+    "ARITHABORT",
+    "ARITHIGNORE",
+    "FMTONLY",
+    "NOCOUNT",
+    "NOEXEC",
+    "NUMERIC_ROUNDABORT",
+    "PARSEONLY",
+    "QUERY_GOVERNOR_COST_LIMIT",
+    "RESULT CACHING (Preview)",
+    "ROWCOUNT",
+    "TEXTSIZE",
+    "ANSI_DEFAULTS",
+    "ANSI_NULL_DFLT_OFF",
+    "ANSI_NULL_DFLT_ON",
+    "ANSI_NULLS",
+    "ANSI_PADDING",
+    "ANSI_WARNINGS",
+    "FORCEPLAN",
+    "SHOWPLAN_ALL",
+    "SHOWPLAN_TEXT",
+    "SHOWPLAN_XML",
+    "STATISTICS IO",
+    "STATISTICS XML",
+    "STATISTICS PROFILE",
+    "STATISTICS TIME",
+    "IMPLICIT_TRANSACTIONS",
+    "REMOTE_PROC_TRANSACTIONS",
+    "TRANSACTION ISOLATION LEVEL",
+    "XACT_ABORT",
```

Comment:
Would it make sense to sort these alphabetically? To make it easier to find if a word has been added or not?

Reply:
Done